src/common/filament.cpp

```diff
@@ -31,15 +30,15 @@ static constexpr const char *flex_str =  "FLEX     240/ 50";
 //not a member - static_assert cannot access private members
 //fixme generating long names, takes too long
```

Comment:
Does this comment still apply?

Reply:
good eye, those comments are no longer valid